components/molecules/AuthSection/auth-section.tsx

```diff
@@ -102,7 +102,7 @@ const AuthSection: React.FC = ({}) => {
           <SearchDialogTrigger />
           {user ? (
             <>
-              {!onboarded && (
+              {onboarded !== undefined && !onboarded && (
```

Comment:
Refactor (non-blocking): We could default to `false` up top, i.e. 

`const { onboarded = false, session } = useSession(true);`

Reply:
`onboarded` is undefined initially, then set to `true/false` after the session loads, which is why I don't want the button to show when its `undefined`